RecoBTag/SecondaryVertex/plugins/TemplatedSecondaryVertexProducer.cc

```diff
@@ -860,9 +860,10 @@ void TemplatedSecondaryVertexProducer<IPTI,VTX>::produce(edm::Event &event,
 		  else {
 		      for(size_t iExtSv = 0; iExtSv < extSecVertex->size(); iExtSv++){
 			 const VTX & extVertex = (*extSecVertex)[iExtSv];
-			 if( Geom::deltaR2( ( position(extVertex) - pv.position() ), jetDir ) > extSVDeltaRToJet*extSVDeltaRToJet || extVertex.p4().M() < 0.3 )
+			 if( Geom::deltaR2( ( position(extVertex) - pv.position() ), (extSVDeltaRToJet > 0) ? jetDir : -jetDir ) > extSVDeltaRToJet*extSVDeltaRToJet || extVertex.p4().M() < 0.3 )
```

Comment:
this looks like a bugfix that affects more than just Deep\* negative taggers.
It's worth to be noted in the PR description.

Reply:
What do you mean here? That it also affects CSV?

Reply:
I'm re-reading the description and now I see that it also mentions "(and CSVv2) ".
OK then.